[docs] for DBZ-7284 (#5150)
debezium/debezium: documentation/modules/ROOT/pages/integrations/cloudevents.adoc

|false
|Specifies whether schema name of CloudEvents' `data` field can be retrieved from a header. The specific name should be in `metadata.source` property
(`dataSchemaName` parameter).

|[[cloud-events-converter-extension-attributes-enable]]xref:cloud-events-converter-extension-attributes-enable[`extension.attributes.enable`]
|`true`
|Specifies whether the converter includes extension attributes when it generates a cloud event. 
The value can be `true` or `false`.

|[[cloud-events-converter-metadata-source]]xref:cloud-events-converter-metadata-source[`metadata.source`]
|`value,id:generate,type:generate,dataSchemaName:generate`
|A comma-separated list that specifies the sources from which the converter retrieves metadata (source, operation, transaction), values for CloudEvent `id` and `type` fields,
and `dataSchemaName` (a parameter that specifies the name under which the schema is registered in a Schema Registry).
The first element in the list is a global setting that specifies the source of the metadata.  
The source of metadata can be `value` or `header`.

> The global setting is followed by a set of pairs the first element of which specifies the name of a CloudEvent field (`id` or `type`) or data schema name parameter (`dataSchemaName`),
> and the second element specifies the source for obtaining the value: `generate` or `header`.
> Separate the values in each pair with a colon, for example:

Reviewer: ```suggestion
The global setting is followed by a set of pairs.
The first element in each pair specifies the name of a CloudEvent field (`id` or `type`), or the name of a data schema (`dataSchemaName`).
The second element in the pair specifies how the converter populates the value of the field.
Valid values are `generate` or `header`.
Separate the values in each pair with a colon, for example:
```
